src/components/clusters/ClustersTable.tsx

```diff
@@ -40,6 +40,8 @@ const columns = [
   { title: 'Version', ...columnConfig },
   { title: 'Status', ...columnConfig },
   { title: 'Hosts', ...columnConfig },
+  { title: 'Status Updated At', ...columnConfig },
+  { title: 'Created At', ...columnConfig },
```

Comment:
If the date format we use is "2 days ago" and eventually a date, `Created` without the `at` may be better here, which is what OCM uses. Their `Created` is a bit different than ours, but I'm struggling to find out a better word.

Maybe our `database_created_date` could be shown as the `Created` date while the cluster is a Draft, and after it succeeds the `installation_finished_time` could be used as the `Created` date instead? WDYT?